users/vieting/experiments/switchboard/ctc/feat/baseline_args.py

```diff
@@ -237,6 +253,40 @@ def get_returnn_config(
         # network["source"] = specaug_layer_jingjing(in_layer=["features"])
         pass
 
+    if audio_perturbation and recognition:
+        if preemphasis_perturbation:
+            for layer in feature_net["subnetwork"]:
+                layer_config = feature_net["subnetwork"][layer]
+                if layer_config.get("class") != "variable":
+                    if feature_net["subnetwork"][layer].get("from") == "data":
+                        feature_net["subnetwork"][layer]["from"] = "preemphasis"
+            feature_net["subnetwork"]["preemphasis"] = PreemphasisNetwork(alpha=0.97).get_as_subnetwork()
```

Comment:
We should maybe introduce a default value in the perturbation params and use it here as well.